src/style.css

```
@@ -0,0 +1,127 @@
+body {
+  margin: 0;
+  font-family: "Roboto";
+  font-weight: 500;
+  background: #e5e5e5;
+}
+
+.card {
+  display: flex;
+  align-items: center;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  width: 200px;
+  height: 403px;
+}
+
+.card-ins {
+  display: flex;
+  flex-direction: column;
+
+}
+
+.imac {
+  position: relative;
+  left: 20px;
+  top: 30px;
+  right: 20px;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 45px;
+}
+
+.star {
+  width: 15px;
+  height: 15px;
+}
+
+.star-review {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 15px 15px 0;
+}
+
+.review {
+  font-family: inherit;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
+  text-align: right;
+  color: #000;
+  margin: 0;
+}
+
+.price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.price-name {
+  font-family: inherit;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
+  text-align: right;
+  color: #000;
+  margin-left: 15px;
+  align-self: center;
+}
+
+.price-sell {
+  font-family: inherit;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+  text-align: right;
+  margin-right: 7.5%;
+  color: #000;
+}
+
+.item-code {
+  font-family: inherit;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
+  color: #000;
+  margin: 6px 0 0 15px;
+}
+
+.description {
+  margin: 36px 15px 0;
+  font-family: inherit;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  color: #000;
+}
+
+.button {
+  display: block;
+  box-sizing: border-box;
+  border-radius: 5px;
+  width: 170px;
+  height: 40px;
+  background-color: #00acdc;
+  color: #fff;
+  text-decoration: none;
```

better to write 
```suggestion
a {
  text-decoration: none;
  }
```
we can have many link on our page